src/components/views/HomeView/HomeView.tsx

```diff
@@ -1,11 +1,20 @@
-import {firebase} from '@react-native-firebase/auth';
-import {View, Text, Button} from 'react-native';
+import {useActor} from '@xstate/react';
+import {View, Text} from 'react-native';
+import auth from '@react-native-firebase/auth';
+import {useContext, useEffect} from 'react';
+import {GlobalStateContext} from '../../../components/molecules';
 
 export const HomeView = () => {
+  const globalServices = useContext(GlobalStateContext);
+  //@ts-ignore
+  const [, send] = useActor(globalServices.userService);
+  const id = auth().currentUser?.uid as string;
```

Comment:
potrzebne to as string? Ta funkcja powinna zwrócić konkretny typ, i ta funkcja powinna być otypowana, a nie, że rzutujesz sobie przy jej użyciu

Reply:
done